src/colors.ts

```diff
@@ -5,7 +5,7 @@ export const lightGrey = '#E8E8E8'
 export const faintGrey = '#f5f5f5'
 export const blue = '#295dc0'
 export const red = '#c30000'
-export const greenOnPurple = '#4FAC40'
-export const redOnPurple = '#D61E42'
+export const greenOnPurple = '#47D232'
+export const redOnPurple = '#FF3C2E'
```

Comment:
@varCepheid These changes impact the boolean stats display on the team@event page.

On this branch:

https://deploy-preview-1522--peregrine.netlify.app/events/2023wamc/teams/360

![image](https://github.com/Pigmice2733/peregrine-frontend/assets/13206945/14884fbd-c5e1-4517-9468-4d37bc7a697e)

On dev:

https://dev.peregrinefrc.com/events/2023wamc/teams/360

![image](https://github.com/Pigmice2733/peregrine-frontend/assets/13206945/66f3305b-8611-4a71-ae76-f6675bd901f3)

I chose the colors that are currently on dev because they maintain enough contrast against the white check mark and x. The colors that you changed them to have a lot less contrast against white, especially the bright green.

Also, it's not clear to me why you changed redOnPurple in this PR since you aren't using it in this PR.